### ISSUE TITLE
Strange error with node_modules

### ISSUE THREAD

**Issue by dsvgl** [danielroe/nuxt-vitest#46]
I have a pnpm monorepo and after updating to the latest v0.6.0 I get this error when running `pnpm i`.
Reproduction (no monorepo): https://stackblitz.com/edit/github-kl3fpk?file=vitest.config.ts,package.json,nuxt.config.ts

```
. prepare: widget nuxi:prepare:  ERROR  Error while requiring module vitest-environment-nuxt/module: Error: Cannot find module '/Users/antfu/i/vitest-environment-nuxt/node_modules/.pnpm/jiti@1.16.2/node_modules/jiti/lib/index.js'
. prepare: widget nuxi:prepare: Require stack:
. prepare: widget nuxi:prepare: - /Users/USERNAME/code/monorepo-test/node_modules/.pnpm/vitest-environment-nuxt@0.6.0_vitest@0.28.2/node_modules/vitest-environment-nuxt/dist/module.cjs
. prepare: widget nuxi:prepare:  ERROR  Cannot find module '/Users/antfu/i/vitest-environment-nuxt/node_modules/.pnpm/jiti@1.16.2/node_modules/jiti/lib/index.js'
. prepare: widget nuxi:prepare: Require stack:
. prepare: widget nuxi:prepare: - /Users/USERNAME/code/monorepo-test/node_modules/.pnpm/vitest-environment-nuxt@0.6.0_vitest@0.28.2/node_modules/vitest-environment-nuxt/dist/module.cjs
. prepare: widget nuxi:prepare:   Require stack:
. prepare: widget nuxi:prepare:   - /Users/USERNAME/code/monorepo-test/node_modules/.pnpm/vitest-environment-nuxt@0.6.0_vitest@0.28.2/node_modules/vitest-environment-nuxt/dist/module.cjs
. prepare: widget nuxi:prepare:   at Function.Module._resolveFilename (node:internal/modules/cjs/loader:956:15)
. prepare: widget nuxi:prepare:   at Function.resolve (node:internal/modules/cjs/helpers:108:19)
. prepare: widget nuxi:prepare:   at _resolve (/Users/USERNAME/code/monorepo-test/node_modules/.pnpm/jiti@1.16.2/node_modules/jiti/dist/jiti.js:1:244204)
. prepare: widget nuxi:prepare:   at jiti (/Users/USERNAME/code/monorepo-test/node_modules/.pnpm/jiti@1.16.2/node_modules/jiti/dist/jiti.js:1:246368)
. prepare: widget nuxi:prepare:   at /Users/USERNAME/code/monorepo-test/node_modules/.pnpm/vitest-environment-nuxt@0.6.0_vitest@0.28.2/node_modules/vitest-environment-nuxt/dist/module.cjs:1:93
. prepare: widget nuxi:prepare:   at jiti (/Users/USERNAME/code/monorepo-test/node_modules/.pnpm/jiti@1.16.2/node_modules/jiti/dist/jiti.js:1:248448)
. prepare: widget nuxi:prepare:   at requireModule (/Users/USERNAME/code/monorepo-test/node_modules/.pnpm/@nuxt+kit@3.1.0/node_modules/@nuxt/kit/dist/index.mjs:286:26)
. prepare: widget nuxi:prepare:   at normalizeModule (/Users/USERNAME/code/monorepo-test/node_modules/.pnpm/@nuxt+kit@3.1.0/node_modules/@nuxt/kit/dist/index.mjs:451:55)
. prepare: widget nuxi:prepare:   at Object.installModule (/Users/USERNAME/code/monorepo-test/node_modules/.pnpm/@nuxt+kit@3.1.0/node_modules/@nuxt/kit/dist/index.mjs:434:47)
. prepare: widget nuxi:prepare:   at setup (/Users/USERNAME/code/monorepo-test/node_modules/.pnpm/nuxt-vitest@0.6.0_jsdom@21.0.0/node_modules/nuxt-vitest/dist/module.cjs:19:15)
. prepare: widget nuxi:prepare: Failed
```

I have a separate nuxi prepare script in each packages package.json. So, nuxi prepare runs on each package. `nuxt-vitest` is only configured in my `widget` package...
Removed all node_module folders and lock file -> same error.
This looks strange: `Cannot find module '/Users/antfu/i/vitest-environment-nuxt/node_modules/.pnpm/jiti@1.16.2/node_modules/jiti/lib/index.js'`

Maybe it'S clear for the authors whats the problem here. Else, I can try to create a reproduction.

**Comment by dsvgl** [comment]
**Update:** created a reproduction: https://stackblitz.com/edit/github-kl3fpk?file=vitest.config.ts,package.json,nuxt.config.ts
This is a simple nuxt app (no monorepo). It throws as soon as I add `modules: ['nuxt-vitest']` into nuxt.config and then do a `npm i`.

**Comment by antfu** [comment]
It was a build bug, and should be fixed in v0.6.1.

**Comment by dsvgl** [comment]
nice, thx!
